modules/pixfutureBidAdapter.js

```diff
@@ -0,0 +1,319 @@
+    const hostname = this.getHostname();
+    return validBidRequests.map((bidRequest) => {
+      let referer = '';
+      if (bidderRequest && bidderRequest.refererInfo) {
+        referer = bidderRequest.refererInfo.referer || '';
+      }
+
+      const userObjBid = find(validBidRequests, hasUserInfo);
+      let userObj = {};
+      if (config.getConfig('coppa') === true) {
+        userObj = {'coppa': true};
+      }
+
+      if (userObjBid) {
+        Object.keys(userObjBid.params.user)
+          .filter(param => includes(USER_PARAMS, param))
+          .forEach((param) => {
+            let uparam = utils.convertCamelToUnderscore(param);
+            if (param === 'segments' && utils.isArray(userObjBid.params.user[param])) {
+              let segs = [];
+              userObjBid.params.user[param].forEach(val => {
+                if (utils.isNumber(val)) {
+                  segs.push({'id': val});
+                } else if (utils.isPlainObject(val)) {
+                  segs.push(val);
+                }
+              });
+              userObj[uparam] = segs;
+            } else if (param !== 'segments') {
+              userObj[uparam] = userObjBid.params.user[param];
+            }
+          });
+      }
+
+      const schain = validBidRequests[0].schain;
+
+      const payload = {
+        tags: [...tags],
+        user: userObj,
+        sdk: {
+          source: SOURCE,
+          version: '$prebid.version$'
+        },
+        schain: schain
+      };
+
+      if (bidderRequest && bidderRequest.uspConsent) {
+        payload.us_privacy = bidderRequest.uspConsent
+      }
+
+      if (bidderRequest && bidderRequest.refererInfo) {
+        let refererinfo = {
+          rd_ref: encodeURIComponent(bidderRequest.refererInfo.referer),
+          rd_top: bidderRequest.refererInfo.reachedTop,
+          rd_ifs: bidderRequest.refererInfo.numIframes,
+          rd_stk: bidderRequest.refererInfo.stack.map((url) => encodeURIComponent(url)).join(',')
+        };
+        payload.referrer_detection = refererinfo;
+      }
+
+      if (validBidRequests[0].userId) {
+        let eids = [];
+
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.flocId.id`), 'chrome.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.criteoId`), 'criteo.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.unifiedId`), 'thetradedesk.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.id5Id`), 'id5.io', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.sharedId`), 'thetradedesk.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.identityLink`), 'liveramp.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.liveIntentId`), 'liveintent.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.fabrickId`), 'home.neustar', null);
```

Comment:
In your docs PR you added `userIds: all`, but here you don't pull **all** userIds, just couple of them. Either correct your docs PR, or add support for all of them.

Reply:
In the docs PR has been fixed list of `userIds`